tests/test_wsgi.py

```diff
@@ -280,3 +253,20 @@ def test_dont_unquote_environ_path_info(self):
             'GET', path, (1, 0), self.headers, True, 'deflate')
         environ = self._make_one()
         self.assertEqual(environ['PATH_INFO'], path)
+
+    def test_not_add_authorization(self):
+        self.headers.extend({"AUTHORIZATION": "spam",
+                             "X-CUSTOM-HEADER": "eggs"})
+        self.message = protocol.RawRequestMessage(
+            'GET', '/', (1, 1), self.headers, True, 'deflate')
+        environ = self._make_one()
+        self.assertEqual('eggs', environ['HTTP_X_CUSTOM_HEADER'])
+        self.assertFalse('AUTHORIZATION' in environ)
+
+    def test_http_1_0_no_host(self):
+        headers = multidict.MultiDict({})
+        self.message = protocol.RawRequestMessage(
+            'GET', '/', (1, 0), headers, True, 'deflate')
+        environ = self._make_one()
+        self.assertEqual('1.2.3.4', environ["SERVER_NAME"])
+        self.assertEqual('8080', environ["SERVER_PORT"])
```

Comment:
Hmmm probably I should also check self.transport.get_extra_info called with proper arguments...